extensions/amazon-lambda-xray/runtime/pom.xml

```
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-amazon-lambda-xray-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>quarkus-amazon-lambda-xray</artifactId>
+    <name>Quarkus - Amazon Lambda Xray - Runtime</name>
+    <description>Modifications to allow Xray to run with Quarkus Lambda support</description>
```

```suggestion
    <description>Allows X-Ray to run with Quarkus Lambda support</description>
```
(this description is the one we get on code.quarkus.io)